src/harness.js

```diff
@@ -659,6 +708,38 @@ const clearSync = (self) => {
   self._now = undefined;
 
   sinon.restore();
+  self.pushMockedDoc(...self.options.inputs.docs);
+};
+
+const resolveContent = async (coreAdapter, state, content, contact) => {
```

Comment:
Are these two functions `resolveContent` and `resolveMock` declared outside of the Harness class so you keep the exported prototype clean?

Reply:
yes. to keep the public interface for the harness clean. any better option?